### PR TITLE
FUSETOOLS-2887: Create a new test for Rest DSL editor

### PR DESCRIPTION
**Pull request by apodhrad**
FUSETOOLS-2887: Create a new test for checking if Rest DSL editor displays Rest settings/methods correctly

Signed-off-by: Andrej Podhradsky <apodhrad@redhat.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/RESTEditor.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/package org.jboss.tools.fuse.reddeer.editor;
```
Comment (tsedmik) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/RESTEditor.java]: move _package ..._ to a new line

**Comment by apupier** [comment]
testPR

**Comment by tsedmik** [comment]
testPR

**Comment by tsedmik** [comment]
testPR

**Comment by lhein** [comment]
There is no valid review / approval so I removed the ready to merge flag

**Pull request merged**